disk/disk_unix.go

```diff
@@ -2,11 +2,11 @@
 
 package disk
 
-import "syscall"
+import "golang.org/x/sys/unix"
 
 func Usage(path string) (*UsageStat, error) {
-	stat := syscall.Statfs_t{}
-	err := syscall.Statfs(path, &stat)
+	stat := unix.Statfs_t{}
```

Comment:
Drat, let me look into that.  I'm mid-way through cherrypicking some patches onto this branch that will add memory support.  I don't know why I didn't catch this during testing, however.

IMNSHO the `syscall` package should _not_ be used anywhere where an `x/sys/unix` package variant is available _because_ the `syscall` package is frozen, whereas the `x/sys/unix` package will track the current API.

In this particular case, there wasn't support for `syscall.Statfs` on the target platform, but was under the `unix.` package.  It has been lamented a few times by the Go authors that the `syscall` package should be removed.  IMO `syscall` would have been "fine" (on some level) if it weren't frozen.  But since `unix` is there and not frozen, `syscall` should be outright deprecated and eschewed from all code bases so that programs can converge on a single maintained package (read: `unix` > `syscall`).

For instance, Illumos grew support for `unix. Statvfs` which didn't exist previously and couldn't have existed under the `syscall` package. 
 https://github.com/golang/sys/commit/1e99a4f9d247b28c670884b9a8d6801f39a47b77